src/test/java/seedu/command/DeleteCategoryCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.category.Category;
+import seedu.category.CategoryList;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class DeleteCategoryCommandTest {
+
+    @Test
+    void execute() {
```

Comment:
The method name convention is wrong. Please check your assigned bug issue for more information